HW03/Alexey_Zalyotov/lib/repo.rb

```diff
@@ -0,0 +1,27 @@
+class Repo
+  attr_accessor :homeworks, :students, :mentors
+
+  def initialize
+    @homeworks = []
+    @mentors = []
+    @students = []
+  end
+
+  def show_homeworks
+    homeworks.empty? ? (puts 'No homeworks') : (puts formated_homeworks)
+  end
+
+  private
+
+  def formated_homeworks
+    homeworks.map do |homework|
+      "____________________________\n" \
+      "Task ##{homeworks.index(homework) + 1}\n" \
+      "id: #{homework.object_id}\n" \
+      "task: #{homework.task}\n" \
+      "status: #{homework.status}\n" \
+      "review_comment: #{homework.review_comment}\n" \
+      "solution: #{homework.solution}"
```

Comment:
I think we can move this logic to the `Homework` class, it's not the responsibility of class `Repo`. What do you think? 
```suggestion
  homework.formated_homework
```

Reply:
Done